addons/website/models/res_config_settings.py

```diff
@@ -14,116 +13,173 @@ class ResConfigSettings(models.TransientModel):
-                                           help='Are newly created user accounts website specific')
-    website_cookies_bar = fields.Boolean(related='website_id.cookies_bar', readonly=False)
-
-    google_analytics_key = fields.Char('Google Analytics Key', related='website_id.google_analytics_key', readonly=False)
-    google_management_client_id = fields.Char('Google Client ID', related='website_id.google_management_client_id', readonly=False)
-    google_management_client_secret = fields.Char('Google Client Secret', related='website_id.google_management_client_secret', readonly=False)
-    google_search_console = fields.Char('Google Search Console', related='website_id.google_search_console', readonly=False)
-
-    cdn_activated = fields.Boolean(related='website_id.cdn_activated', readonly=False)
-    cdn_url = fields.Char(related='website_id.cdn_url', readonly=False)
-    cdn_filters = fields.Text(related='website_id.cdn_filters', readonly=False)
-    auth_signup_uninvited = fields.Selection(compute="_compute_auth_signup", inverse="_set_auth_signup")
-
-    social_twitter = fields.Char(related='website_id.social_twitter', readonly=False)
-    social_facebook = fields.Char(related='website_id.social_facebook', readonly=False)
-    social_github = fields.Char(related='website_id.social_github', readonly=False)
-    social_linkedin = fields.Char(related='website_id.social_linkedin', readonly=False)
-    social_youtube = fields.Char(related='website_id.social_youtube', readonly=False)
-    social_instagram = fields.Char(related='website_id.social_instagram', readonly=False)
-
-    @api.depends('website_id', 'social_twitter', 'social_facebook', 'social_github', 'social_linkedin', 'social_youtube', 'social_instagram')
-    def has_social_network(self):
-        self.has_social_network = self.social_twitter or self.social_facebook or self.social_github \
-            or self.social_linkedin or self.social_youtube or self.social_instagram
-
-    def inverse_has_social_network(self):
-        if not self.has_social_network:
-            self.social_twitter = ''
-            self.social_facebook = ''
-            self.social_github = ''
-            self.social_linkedin = ''
-            self.social_youtube = ''
-            self.social_instagram = ''
-
-    has_social_network = fields.Boolean("Configure Social Network", compute=has_social_network, inverse=inverse_has_social_network)
-
-    favicon = fields.Binary('Favicon', related='website_id.favicon', readonly=False)
-    social_default_image = fields.Binary('Default Social Share Image', related='website_id.social_default_image', readonly=False)
-
-    google_maps_api_key = fields.Char(related='website_id.google_maps_api_key', readonly=False)
-    group_multi_website = fields.Boolean("Multi-website", implied_group="website.group_multi_website")
-
-    @api.onchange('website_id')
+    website_id = fields.Many2one(
```

Comment:
Waaaaay too much diff that breaks forward ports, history,...

Reply:
I agree for forward ports, but res.config fixes are not done on a daily basis too, and are most likely to start from this version.